tox.ini

```diff
@@ -30,9 +32,17 @@ allowlist_externals =
 env_dir={toxworkdir}/acceptance
 commands =
     behave {posargs}
+### Uncomment following lines when running webrtc playwright tests in headed mode
+# pass_env =
+#    XAUTHORITY
+#    DISPLAY
 
 [testenv:linters]
 base_python = python3.10
 skip_install = true
 deps = pre-commit
 commands = pre-commit run --all-files
+
+[behave]
+default_tags = not (@transcoding)
```

Comment:
Debating if we shouldn't use a flag in the wazo-acceptance config file to control this. 
Thinking out loud, but maybe something along the lines of `has_rtpengine = true/false` in the instance or something? Would feel more descriptive of why the test  is there and enabled or not because of the testing environment